src/utils/fetchClient.ts

```diff
@@ -26,7 +24,7 @@ function request<T>(
     };
   }
 
-  // for a demo purpose we emulate a delay to see if Loaders work
+  // should be 300
```

Comment:
The comment 'should be 300' is unclear. It might refer to the delay in the `wait` function, but it's not explicit. Consider clarifying the comment to indicate its purpose or remove it if it's unnecessary.